airflow/providers/ssh/operators/ssh.py

```diff
@@ -151,16 +151,18 @@ def exec_ssh_client_command(self, ssh_client: SSHClient, command: str):
             ssh_client, command, timeout=self.timeout, environment=self.environment, get_pty=self.get_pty
         )
 
-    def raise_for_status(self, exit_status: int, stderr: bytes) -> None:
+    def raise_for_status(self, exit_status: int, stderr: bytes, **kwargs) -> None:
+        ti=kwargs["context"].get("task_instance")
+        ti.xcom_push(key="ssh_exit", value=exit_status)
```

Comment:
I think we should verify user wish to push to xcom.
users ask for this specifically with `do_xcom_push` parameter

Reply:
@eladkal It is not optional on [other operators](https://github.com/apache/airflow/blob/main/airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py#L444).